contrib/fetch-offline-data/instana/README.md

```diff
@@ -68,4 +68,55 @@ python instana_fetcher.py
 python main.py --ingest_type=file --ingest_file=../contrib/fetch-offline-data/instana/promql_metrics.json --feature_extraction_type=tsfel --config_generator_type=otel --config_generator_directory=/tmp
 ```
 
+# INSTANA Infrastructure metrics
```

Comment:
Thank you for the expanded functionality of the fetch operations.
Does the section on INSTANA Infrastructure metrics supersede the previous sections concerning the program arguments? For example, the earlier description has a --limit argument, which does not appear here. Or does this section simply apply to a different command? The examples below use `fetch_instana_data.py`.

Reply:
I removed limit argument as it is preventing the code from fetching relevant metric. It is also not very straight forward for the user to determine a specific limit before running the code to determine the number which can suffice user requirement.